src/EnergyPlus/DXCoils.cc

```diff
@@ -9955,7 +9956,13 @@ Label50: ;
 				}
 			}
 			ShowContinueErrorTimeStamp( "" );
-			ShowFatalError( "Check and revise the input data for this coil before rerunning the simulation." );
+			ShowContinueError( "SHR adjusted to achieve valid outlet air properties and the simulation continues." );
+			OutletAirTempSat = PsyTsatFnHPb(OutletAirEnthalpy, BaroPress, RoutineName);
+			if (OutletAirTemp < OutletAirTempSat) { // Limit to saturated conditions at OutletAirEnthalpy
+				OutletAirTemp = OutletAirTempSat + 0.005;
+				OutletAirHumRat = PsyWFnTdbH( OutletAirTemp, OutletAirEnthalpy, RoutineName );
+				DeltaHumRat = InletAirHumRat - OutletAirHumRat;
+			}
```

Comment:
Is there a similar `DeltaTemp` variable that needs to be updated here as well since you are resetting the `OutletAirTemp` variable?  Might be done below, I'm just commenting as I go.